core/queueref.go

```diff
@@ -1,9 +1,21 @@
 package core
 
 import (
+	"errors"
+	"sync"
 	"unsafe"
 )
 
+var (
+	MaxQueueSize = 10000
+	MaxQueues    = 1000
+)
```

Comment:
These should be a part of the config instead of being global variables.

Reply:
Done.